projects/schematics/src/add-spartacus/index.ts

```diff
@@ -343,7 +365,115 @@ export function addSpartacus(options: SpartacusOptions): Rule {
       installStyles(options),
       updateMainComponent(project, options),
       options.useMetaTags ? updateIndexFile(tree, options) : noop(),
+
+      addSpartacusFeatures(options),
+
       installPackageJsonDependencies(),
     ])(tree, context);
   };
 }
+
+function addSpartacusFeatures(options: SpartacusOptions): Rule {
```

Comment:
Can we do it an a way to avoid multiple installs?

Reply:
Not sure what do you mean?